public/catch-js-errors.js

```diff
@@ -108,8 +111,8 @@
 			}
 		}
 
-		if ( localStorage.getItem( 'log-errors' ) !== undefined &&
-			( localStorage.getItem( 'log-errors' ) === 'rest-api' || localStorage.getItem( 'log-errors' ) === 'true' )
+		if ( localStorageAssignment !== undefined &&
```

Comment:
Why do we need to check for undefined? We're doing a equality check after this that shouldn't throw any errors.